lgc/patch/NggPrimShader.cpp

```diff
@@ -666,12 +666,12 @@ void NggPrimShader::constructPrimShaderWithoutGs(Module *module) {
         //   ES_GS_OFFSET01[8:0]   = vertexId0 (in bytes)
 
         // Distribute primitive ID
-        auto vertexId0 = CreateUBfe(m_nggFactor.primData, 0, 9);
-        writePerThreadDataToLds(gsPrimitiveId, vertexId0, LdsRegionDistribPrimId);
-        auto vertexId1 = CreateUBfe(m_nggFactor.primData, 10, 9);
-        writePerThreadDataToLds(gsPrimitiveId, vertexId1, LdsRegionDistribPrimId);
-        auto vertexId2 = CreateUBfe(m_nggFactor.primData, 20, 9);
-        writePerThreadDataToLds(gsPrimitiveId, vertexId2, LdsRegionDistribPrimId);
+        Value *vertexId = nullptr;
```

Comment:
It look like `provokingVertexMode` only changes the hash in `updateHashForFragmentState`; however this will modify the non-fragment shader stages right?

You should probably be including it here:

https://github.com/GPUOpen-Drivers/llpc/blob/7c02878905ce7e0cf6a3dece5965c4102fcc10fe/tool/dumper/vkgcPipelineDumper.cpp#L1003-L1009

Reply:
Fixed, Thanks.